Revised IVA verification instructions (#99)
ghga-de/docs.ghga.de: user_docs/other/iva.md

### How to Verify an IVA

>  GHGA offers two ways to add IVAs (Mobile Phones and in Person). A verification address can be added in the [GHGA Data Portal](https://data.ghga.de/). 

Reviewer: ```suggestion
 GHGA offers two types of IVAs:  Mobile Phones and in Person validation. A new IVA can be added to your profile in the [GHGA Data Portal](https://data.ghga.de/). 
```
To make it clearer

*For practicability mobile phones are recommended but please note that <general:Research Data Controller (RDC)> might demand alternative means during negotiation of a <general:Data Transfer Agreement (DTA)>*.

1. To create and verify a contact navigate to the [GHGA Data Portal](https://data.ghga.de/).

> 2. After logging in visit "Your GHGA account page" by clicking on your profile on the top right corner.

Reviewer: ```suggestion
2. After logging in, visit "Your GHGA account page" by clicking on your profile on the top right corner.
```

    ![Screenshot of GHGA Data Portal profile page showing where to add a new IVA – as described in the text above.](../assets/img/add_iva.png){ width="800" }

4. Select an IVA type from SMS, or In-Person. Please note, in person verification is only offered for personnel located at a GHGA Data Hub. The fastest verification can be performed via SMS.

    ![Screenshot of contact method selection for account verification in the GHGA Data Portal – SMS or In-Person – as described in the text above.](../assets/img/iva_select.png){ width="800" }

5. Enter the required information for the chosen type and confirm them by clicking on "Submit".

    ![Screenshot of GHGA Portal showing how to enter and add an unverified contact address – as explained in the text above.](../assets/img/address_submit.png){ width="800" }

6. The menu "Independent Verification Addresses (IVAs)" will add the unverified address. A code for this address can be requested by clicking "Request verification".

    ![Screenshot of GHGA Portal showing how to request verification for a newly added contact address – as described in the text above.](../assets/img/request_verification.png){ width="800" }

> 7. A code will be transmitted if "SMS" was selected. You will also be notified via mail when a code has been transmitted. Upon receival of the code, click on "Enter verification code", enter the code and confirm the transmission. 

Reviewer 2: ```suggestion
7. A code will be transmitted if "SMS" was selected. Upon receival of the code, click on "Enter verification code", enter the code and confirm the transmission. 
```
@tanavader0 does the portal still send mail confirmations when a code is created? I'm not 100% sure anymore. If it does, just discard the suggestion!

Author: The mail does not contain the code. Although it does send a confirmation that a code has been transmitted. But this was when I did it maybe 6 months back. Not sure if there were changes after.
